src/datetime/mod.rs

```diff
@@ -1530,6 +1527,66 @@ where
     }
 }
 
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_add_sub_months() {
+        let utc_dt = Utc.with_ymd_and_hms(2018, 9, 5, 23, 58, 0).unwrap();
+        assert_eq!(utc_dt + Months::new(15), Utc.with_ymd_and_hms(2019, 12, 5, 23, 58, 0).unwrap());
+
+        let utc_dt = Utc.with_ymd_and_hms(2020, 1, 31, 23, 58, 0).unwrap();
+        assert_eq!(utc_dt + Months::new(1), Utc.with_ymd_and_hms(2020, 2, 29, 23, 58, 0).unwrap());
+        assert_eq!(utc_dt + Months::new(2), Utc.with_ymd_and_hms(2020, 3, 31, 23, 58, 0).unwrap());
+
+        let utc_dt = Utc.with_ymd_and_hms(2018, 9, 5, 23, 58, 0).unwrap();
+        assert_eq!(utc_dt - Months::new(15), Utc.with_ymd_and_hms(2017, 6, 5, 23, 58, 0).unwrap());
+
+        let utc_dt = Utc.with_ymd_and_hms(2020, 3, 31, 23, 58, 0).unwrap();
+        assert_eq!(utc_dt - Months::new(1), Utc.with_ymd_and_hms(2020, 2, 29, 23, 58, 0).unwrap());
+        assert_eq!(utc_dt - Months::new(2), Utc.with_ymd_and_hms(2020, 1, 31, 23, 58, 0).unwrap());
+    }
+
+    #[test]
+    fn test_auto_conversion_fixedoffset_into_utc() {
```

Comment:
Is this a simpler way to cover the same as this test and the following?
```rust
#[cfg(feature = "clock")]
#[test]
fn test_auto_conversion() {
    let dt = NaiveDate::from_ymd_opt(2018, 9, 5).unwrap().and_hms_opt(23, 58, 0).unwrap();
    let dt_utc = Utc.from_utc_datetime(&dt);
    let dt_fixedoffset = FixedOffset::west_opt(5 * 60 * 60).unwrap().from_utc_datetime(&dt);
    let dt_local = Local.from_utc_datetime(&dt);

    assert_eq!(dt_utc, DateTime::<Utc>::from(dt_fixedoffset));
    assert_eq!(dt_utc, DateTime::<Utc>::from(dt_local));
    assert_eq!(dt_fixedoffset, DateTime::<FixedOffset>::from(dt_utc));
    assert_eq!(dt_fixedoffset, DateTime::<FixedOffset>::from(dt_local));
    assert_eq!(dt_local, DateTime::<Local>::from(dt_utc));
    assert_eq!(dt_local, DateTime::<Local>::from(dt_fixedoffset));
}
```

Reply:
It is considered best practice to prefer many small tests exercising fewer things rather than fewer larger tests exercising many things.